onebusaway-android/src/main/java/org/onebusaway/android/ui/FeedbackActivity.java

```diff
@@ -0,0 +1,168 @@
+            sendLogs.setChecked(false);
+        }
+
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // Inflate the menu; this adds items to the action bar if it is present.
+        getMenuInflater().inflate(R.menu.report_issue_action, menu);
+        return true;
+    }
+
+   @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+       if (item.getItemId() == R.id.report_problem_send) {
+           submitFeedback();
+           finish();
+       }
+        return super.onOptionsItemSelected(item);
+    }
+
+    private void submitFeedback() {
+        PreferenceUtils.saveBoolean(NavigationServiceProvider.FIRST_FEEDBACK, false);
+        Log.d(TAG, "First Feedback : " + String.valueOf(Application.getPrefs()
+                .getBoolean(NavigationServiceProvider.FIRST_FEEDBACK, true)));
+        String feedback = ((EditText) this.findViewById(R.id.editFeedbackText)).getText().toString();
+        if (Application.getPrefs()
+                .getBoolean(getString(R.string.preferences_key_user_share_logs), true)) {
+            moveLog(feedback);
+        }
+        else {
+            deleteLog();
+        }
+        Log.d(TAG,"Feedback send : " + feedback);
+        Toast.makeText(FeedbackActivity.this,
+                getString(R.string.feedback_notify_confirmation),
+                Toast.LENGTH_SHORT).show();
+    }
+
+    public void likeBtnOnClick(View view) {
+        mAction = "yes";
+        likeButton = findViewById(R.id.ImageBtn_like);
+        dislikeButton = findViewById(R.id.ImageBtn_Dislike);
+        likeButton.setSelected(true);
+        dislikeButton.setSelected(false);
+        Log.d(TAG,"Feedback changed to yes");
+    }
+
+    public void dislikeBtnOnClick(View view) {
+        mAction = "no";
+        likeButton = findViewById(R.id.ImageBtn_like);
+        dislikeButton = findViewById(R.id.ImageBtn_Dislike);
+        dislikeButton.setSelected(true);
+        likeButton.setSelected(false);
+        Log.d(TAG,"Feedback changed to no");
+    }
+
+    private void moveLog(String feedback) {
+        try {
+            File lFile = new File(NavigationService.LOG_FILE);
+            FileUtils.write(lFile, System.getProperty("line.separator") + "User Feedback - " + feedback, true);
+            Log.d(TAG, "Feedback appended");
+
+            File destFolder = new File(Application.get().getApplicationContext().getFilesDir()
+                    .getAbsolutePath() + File.separator + LOG_DIRECTORY + File.separator + mAction);
+
+            Log.d(TAG, "sourceLocation: " + lFile);
+            Log.d(TAG, "targetLocation: " + destFolder);
+
+            try {
+                FileUtils.moveFileToDirectory(
+                        FileUtils.getFile(lFile),
+                        FileUtils.getFile(destFolder), true);
+                Log.d(TAG, "Move file successful.");
+            }
+            catch (Exception e) {
+                Log.d(TAG, "File move failed");
+            }
```

Comment:
Looks like there is an extra bracket `}` here?

Reply:
We had to include two try-catch blocks - one to append the user free text comment to the log file and the second one to move the file to 'yes'/'no' device folder. There were some misplaced alignments in the '}'. I have fixed the alignments. Please let me know if it looks ok.